data-hibernate-jpa/src/test/groovy/io/micronaut/data/hibernate/PostgresDbInit.java

```diff
@@ -38,7 +38,7 @@ public BasicJdbcConfiguration onCreated(BeanCreatedEvent<BasicJdbcConfiguration>
         try {
             try (Connection connection = DriverManager.getConnection(configuration.getUrl(), info)) {
                 try (CallableStatement st = connection.prepareCall("""
-CREATE PROCEDURE add1(IN myInput integer, OUT myOutput integer)
+CREATE OR REPLACE PROCEDURE add1(IN myInput integer, OUT myOutput integer)
```

Comment:
Unrelated to this PR, just wanted to get rid of errors I was seeing when procedure is already created. I think for tests it is safe to use `CREATE OR REPLACE`